impeller/renderer/pipeline_builder.h

```diff
@@ -16,6 +16,15 @@
 
 namespace impeller {
 
+class NonRenderingFragment {
+ public:
+  static constexpr std::string_view kLabel = "NonRendering";
+  static constexpr std::string_view kEntrypointName = "non_rendering_main";
+  static constexpr ShaderStage kShaderStage = ShaderStage::kFragment;
+  static constexpr std::string_view kGeneratorName = "NonRendering";
```

Comment:
TODO: right now the debug info comes from the fragment name but for non-rasterizing pipelines it should come from the vertex name.